tests/generator/test_wifis.py

```diff
@@ -604,6 +604,29 @@ def test_wifi_adhoc(self):
 mode=adhoc
 '''})
 
+    def test_wifi_adhoc_wpa(self):
```

Comment:
To get full coverage you probably just need to implement the same test for `band: 5GHz`. Then you could call this test `test_wifi_adhoc_wpa_24ghz` and the other one `...5ghz`.

Reply:
i added a unit test for 5GHz now too.